### PR TITLE
Issue 2000 - Anax ppc64le support feature

### LINKED ISSUE #2000: Anax ppc64le support feature
Here is the set of anax repo changes we want to prepare to include into this update:

Scripts anax-image-ppc64le and agbot-image-ppc64le were added into anax Makefile for the corresponding images output. Registry push operation was not touched.

Dockerfiles for agbot and agent images were added. Some changes regarding library dependencies (like jq) installation were fixed to be performed at UBI8 image.

EPEL repo file with ppc64le dependencies was included for Docker image build process.

### PR DESCRIPTION
**Pull request by jacob-rogers**
Signed-off-by: Dmitry Gorbachev <dmitry.gorbachev@ru.ibm.com>

Resolves #2000

### CODE REVIEW

**Comment by jacob-rogers** [comment]
#2001

**Comment by jacob-rogers** [comment]
@dabooz This PR is ready to be reviewed, thanks in advance!

**Review by dabooz — CHANGES_REQUESTED** [review]

File: anax-in-container/Dockerfile_agbot.ubi.ppc64el [hunk 1]
```diff
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```
[the remark this answers is not in the document]
Comment (jacob-rogers) [reply anax-in-container/Dockerfile_agbot.ubi.ppc64el]: Ok, I got it removed - only agent environment is going to be updated.

File: Makefile [hunk 2]
```diff
@@ -288,25 +287,25 @@ macpkginfo:
...
 	  cp -f $(LICENSE_FILE) $(ANAX_CONTAINER_DIR); \
-	  cd $(ANAX_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_IMAGE_LABELS) -t $(ANAX_IMAGE) -f Dockerfile.ubi . && \
+	  cd $(ANAX_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_IMAGE_LABELS) -t $(ANAX_IMAGE) -f Dockerfile.ubi.$(arch) . && \
 	  docker tag $(ANAX_IMAGE) $(ANAX_IMAGE_STG); \
 	else echo "Building the anax docker image is not supported on $(arch)"; fi
 
 agbot-image:
 	@echo "Producing agbot docker image $(AGBOT_IMAGE)"
-	if [[ $(arch) == "amd64" ]]; then \
+	if [[ $(arch) == "amd64" || $(arch) == "ppc64el" ]]; then \
```
[the remark this answers is not in the document]
Comment (jacob-rogers) [reply Makefile]: Fixed.

**Review by dabooz — CHANGES_REQUESTED** [review]
The file rename of anax-in-container/Dockerfile_agbot.ubi.amd64 needs to be reverted also.

**Comment by jacob-rogers** [comment]
> The file rename of anax-in-container/Dockerfile_agbot.ubi.amd64 needs to be reverted also.

@dabooz Reverted that, thanks.

**Review by dabooz — APPROVED** [review]

**Pull request merged**